relay/polkadot/src/lib.rs

```diff
@@ -1673,6 +1673,137 @@ construct_runtime! {
 	}
 }
 
+#[cfg(feature = "zombie-bite-sudo")] // FAIL-CI
```

Comment:
So we enable this feature by default now?

Reply:
no, is not enabled by default we should add to the just _recipe_ in ahm-dryrun.